includes/payments/class-payments-query.php

```diff
@@ -633,7 +634,6 @@ private function remap_args() {
 
 		if ( isset( $this->args['nopaging'] ) && true === $this->args['nopaging'] ) {
 			// Setting to a really large number because we don't actually have a way to get all results.
-			// @todo Maybe trigger doing_it_wrong.
 			$arguments['number'] = 99999;
```

Comment:
```suggestion
			$arguments['number'] = 9999999;
```

Just to match the number used elsewhere.